baus/ual.py

```diff
@@ -944,10 +949,10 @@ def hlcm_owner_simulate_no_unplaced(households, residential_units,
                                     hlcm_owner_no_unplaced_config):
 
     # only run in the last year, but make sure to run before summaries
```

Comment:
Shall we remove this comment? Or add a new comment explaining why this is no longer the case - which is consistent with commenting out the next two lines.

Reply:
Got it! Thank you.